### DISCUSSION TITLE
Latest privacy protections.

### DISCUSSION THREAD

**Discussion by JuampiVortex** [osudroid/osu-droid#462] [Feature Request] [answered]
I recently switched to a device with a newer version of Android. When I tried to install Osudroid (by the way, great work!), I received this message. I wanted to know if, over time, you could fix it or provide me with a solution.

I analyzed the APK package with VirusTotal and didn't notice anything suspicious. The only issue occurs during installation, as Google Play Services block the installation.

![Screenshot_20241202_191336_Google](https://github.com/user-attachments/assets/4b25eaf8-1200-42cd-828d-62649ab19fb7)

**Comment by Acivev** [comment, marked as answer]
Since this app isn’t from the Play Store, Google Play Protect might flag it. Unfortunately, there’s not much we can do about that, I suppose.

**Reply by JuampiVortex** [reply]
I understand. Honestly, you're doing an amazing job with this game. I've been playing it for a long time, and I never get tired of it. Thank you very much for your response. Best regards and a hug.

**Reply by Acivev** [reply]
First of all, I’m not the only one working on this project. You shouldn’t just thank me! xD

**Reply by JuampiVortex** [reply]
I'm sorry, my friend. It's just that the translator, you know, sometimes translates things differently, and well, it took it as if it were just me. But no, I actually wrote to thank you for your work.

**Reply by Acivev** [reply]
its fine dont worry ^^

**Reply by JuampiVortex** [reply]
Pero bueno, les doy muchas gracias a todos por seguir trabajando en este proyecto. Un saludo y un abrazo para todos.

**Reply by Reco1I** [reply]
> Pero bueno, les doy muchas gracias a todos por seguir trabajando en este proyecto. Un saludo y un abrazo para todos.

Hola. Desde ya el equipo te dice muchas gracias por el apoyo!
